docs/stencil-docs/reference-docs/handlebars-helpers-reference.mdx

```diff
@@ -286,6 +289,60 @@ The following script is an example of Early Hints usage in the Cornerstone theme
 
 You can view more Early Hints usage in Cornerstone's `templates/layout/base.html` file by visiting the [Cornerstone Repository](https://github.com/bigcommerce/cornerstone/commit/f70fcd502343503969c2e475b777a85347137542#).
 
+### EarlyHint
+
+```handlebars showLineNumbers
+{{ earlyHint '{path}' '{REL}' [as='{AS}'] [crossorigin='{CORS}']}}
+```
+
+Retrieves the provided URL and produces an early hint to communicate important web page resources. The server indicates which site assets (such as CSS files, JS scripts, and fonts) a browser needs to render a page fully. When used in conjunction with the `{{cdn}}` helper, load times can improve by 30%.
+
+#### Parameters
+
+- `path` (String): The provided path used when determining web page resources.
+- `REL` (String): Valid values are `preload`, `preconnect`, `prerender`,  and `dns-prefetch`.
+- `AS` (String): Optional. Valid values are `script`, `style`, `font`.
+- `CORS` (String): Optional. Valid values are `no` (default), `anonymous`, and `use-credentials`.

```

Comment:
```suggestion
- `crossorigin` (String): Optional. Valid arguments are `no` (default), `anonymous`, and `use-credentials`.

```

Reply:
This `Parameters` section confuses me because we provide an example of how the helper looks like with all its params (the snippet in lines 294-296), however, in this section we changed the placeholders: `AS`, and `CORS` by its attribute name. To make this more straightforward I'd change the aforementioned example by 

```
{{ earlyHint '{path}' '{rel}' [as='{script|style|font}'] [crossorigin='{no|anonymous|use-credentials}'] }}
```
Furthermore, I'd accept this suggestion above.